src/Sparrow/Json/JsonContextPoolBase.cs

```diff
@@ -417,7 +435,10 @@ public virtual void Dispose()
         public void LowMemory()
         {
             if (LowMemoryFlag.Raise())
+            {
+                Generation++;
```

Comment:
Safer maybe to do Interlocked.Increment?

Reply:
Yes, that would be good to do

Reply:
done